gobblin-service/src/main/java/org/apache/gobblin/service/modules/core/GobblinServiceGuiceModule.java

```diff
@@ -248,7 +262,12 @@ public void configure(Binder binder) {
 
     if (serviceConfig.isWarmStandbyEnabled()) {
       binder.bind(SpecStoreChangeMonitor.class).toProvider(SpecStoreChangeMonitorFactory.class).in(Singleton.class);
-      binder.bind(DagActionStoreChangeMonitor.class).toProvider(DagActionStoreChangeMonitorFactory.class).in(Singleton.class);
+      if (serviceConfig.isDagProcessingEngineEnabled()) {
+        binder.bind(DagActionStoreChangeMonitor.class)
+            .toProvider(DagProcEngineEnabledDagActionStoreChangeMonitorFactory.class).in(Singleton.class);
```

Comment:
did we not update the name of the factory to align w/ the`DagManagementDagActionStoreChangeMonitor` it creates?

if not, shall we now bring them into alignment?